ts/src/bybit.ts

```diff
@@ -5998,41 +5996,61 @@ export default class bybit extends Exchange {
             };
             response = await this.privatePostV5AccountSetMarginMode (this.extend (request, params));
         } else {
-            const market = this.market (symbol);
-            let type = undefined;
-            [ type, params ] = this.getBybitType ('setMarginMode', market, params);
-            if (type === 'linear') {
-                if (isUnifiedAccount) {
-                    throw new NotSupported (this.id + ' setMarginMode() with symbol Unified Account only support inverse contract');
-                }
-                const isUsdtSettled = market['settle'] === 'USDT';
-                if (!isUsdtSettled) {
-                    throw new NotSupported (this.id + ' setMarginMode() with symbol only support USDT perpetual / inverse contract');
+            market = this.market (symbol);
```

Comment:
@sc0Vu we're assuming here that the symbol is always available which is not true

Reply:
@carlosmiei I thought we throw error in market function if symbol is not found...